tests/snuba/rules/conditions/test_event_frequency.py

```diff
@@ -311,7 +311,7 @@ def make_session(i):
                 duration=None,
                 errors=0,
                 # The line below is crucial to spread sessions throughout the time period.
-                started=received - i,
+                started=received - i - 1,
```

Comment:
We had to do this since the new metrics based implementation has the `end` interval as non inclusive so we want data to fit within the interval because we use `interval: x` in tests which is going back `x` seconds from the current time.